examples/streaming/kafka/src/main/scala/org/apache/gearpump/streaming/examples/kafka/dsl/KafkaReadWrite.scala

```diff
@@ -21,8 +21,8 @@ package org.apache.gearpump.streaming.examples.kafka.dsl
 import java.util.Properties
 
 import org.apache.gearpump.cluster.client.ClientContext
-import org.apache.gearpump.cluster.main.{CLIOption, ArgumentsParser}
-import org.apache.gearpump.streaming.dsl.StreamApp
+import org.apache.gearpump.cluster.main.{ArgumentsParser, CLIOption}
+import org.apache.gearpump.streaming.dsl.scalaapi.StreamApp
```

Comment:
dsl packages are restructured into `dsl.api` for common interfaces, `dsl.scalaapi` for Scala and `dsl.javaapi` for Java.